app/sprinkles/admin/src/Controller/GroupController.php

```diff
@@ -524,8 +524,7 @@ public function getUsers(Request $request, Response $response, $args)
 
         // Be careful how you consume this data - it has not been escaped and contains untrusted user-supplied content.
         // For example, if you plan to insert it into an HTML DOM, you must escape it on the client side (or use client-side templating).
-        return $sprunje->toResponse($response);
-    }
+        return $sprunje->toResponse($response);    }
```

Comment:
Style on this line seams off

Reply:
Rather than play whack-a-mole with code styles I ran PHP CS Fixer. Normalised some things from previous commits.

Have to say, not a big fan of the PHPDoc rules. Namely the alignment. Makes internal documentation necessarily tedious, and the added spacing gets a little crazy when exception types are involved.